pkg/controllers/controller.go

```diff
@@ -1,9 +1,15 @@
 package controller
 
 import (
+	"fmt"
 	"time"
 
 	clusterinfov1beta1 "github.com/stolostron/cluster-lifecycle-api/clusterinfo/v1beta1"
+	"github.com/stolostron/multicluster-controlplane/pkg/controllers/addons"
```

Comment:
Please keep the self module import as a separated section.

Reply:
done